### DISCUSSION TITLE
Proceso Legal??🧐

### DISCUSSION THREAD

**Discussion by jcpc91** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#86] [Q&A]
en este punto dice "Proceso Legal"

- Para dar seguimiento a su demanda, el cliente crea una cuenta en la plataforma y verá el seguimiento de cada una de las actualizaciones del proceso legal.

Alguién sabe cual es el proceso legal me imagino que deben ser pasos A -> B -> C pero no tengo ni idea cual es ese proceso

**Comment by GloriaCoralCerecedo** [comment]
Yo investigue el proceso legal o algo así, procesal civil o procesal es algo así creo. Son como 5 pasos, subí por aquí esa info. Y creo que el abogado le va a dar a cliente solo comentarios tipo, "Su demanda está válida", " el proceso tributario está listo" o algo  así por ejemplo. Igual no sé bien eso.

**Reply by patricia2454** [reply]
Saludos, estudio derecho, el proceso legal, sea de la materia que sea (civl, mercantil o penal) implica una cantidad enorme de documentación; pruebas, análisis, resoluciones (según aplique al caso) yo solo puse dentro del diseño web, "consultoría jurídica", "elaboración de contratos" y "consulta de expediente" (momento procesal); sí el futuro cliente solicita una consulta, da click a la forma de pago y se le agenda una entrevista, si necesita la elaboración de un contrato, lo mismo, en el caso de consulta de expediente, llena el formulario paga la consulta y se le da una cita para una respuesta detallada de la situación.

**Reply by patricia2454** [reply]
En caso de necesitar conocer la respuesta del juez en algun problema en específico, (positiva o negativa) ahí mismo, en el botón de "consulta" le arrojara el resultado, tras el pago.

**Reply by GloriaCoralCerecedo** [reply]
Oh en mi caso estaba viendo más de las demandas civiles que son más comunes, para que se haga una demanda pide información del tributario o algo así, del demandante, el demandado, los hechos y pruebas y la petición. Y el proceso son como 5, es como el juicio o algo así, donde van a un juzgado, a lo mejor para no poner tanto. Se puede elegir un proceso predefinido común que ya este, se le agrega un status y se le pone comentario así como mencione "Su demanda se esta revisando", "Su demanda se aprobó", "El día tal tiene cita en el juzgado" y hasta ahí ya lo demás del juzgado ya no concierne.

**Reply by patricia2454** [reply]
Muy bien, si quisieras estructurarlo conforme a derecho, puedes revisar como es el procedimiento que mencionas, revisas el Código Civil Federal y el Código Nacional de Procedimientos Penales, ahí encontraras a detalle los pasos a seguir; pero sólo si deseas extenderte en tus conocimientos. Saludos.

**Comment by Ingrzs** [comment]
Pues yo puse asi ejemplo
Demanda.                   estatus

Demanda robo.    En proceso
Demanda               faltan doc 
Intrafamiliar
Demanda casa.     Concluida

Y abajito le puse comentarios su Demanda ha sido procesada ,carece de datos etc, no ha sido aprobada, concluida, tiene citatorio, el citatorio fue enviado ala persona x presentarse x dia

En status le pones lo que se te ocurra
La cuestión es que les des al cliente como va su Demanda.

**Reply by GloriaCoralCerecedo** [reply]
Oh ya si así es, oh igual tiene un juicio tal día cosas así. Es que yo investiguen unas etapas procesales.

**Comment by martinGM05** [comment]
De acuerdo al ejercicio es mejor hacerlo de manera generíca ya que hay miles de tipos de procesos dependientes y pues es preferible llevar el status con un nombre que el abogado pueda estar editando así como un tipo chat para los comentarios. (Tipo seguimiento de paquetes)

**Reply by GloriaCoralCerecedo** [reply]
Podría ser que tenga una lista de procesos predefinidos, y solo actualice ponga status y comentarios. Y si que como de las paqueterías o algo así

**Reply by mgonzalez94** [reply]
Si, yo también me iria por esa vía, únicamente un listado de estatus genericos del proceso que marquen el inicio-fin del proceso, además de algunos extras para identificar en que momento debe interactuar el cliente o abogado. Y obviamente dentro de los comentarios detallar más sobre el proceso legal algo meramente informativo.
